litmus-portal/backend/graphql-server/pkg/database/mongodb/usermanagement.go

```diff
@@ -0,0 +1,56 @@
+package database
+
+import (
+	"context"
+	"log"
+
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/graph/model"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var userCollection *mongo.Collection
+
+// InsertUser ...
+func InsertUser(ctx context.Context, user *model.User) error {
```

Comment:
Create a separate struct for mongodb

Reply:
Yes we need to have seperate structs. But that will be needed if our mongodb schema is separate from graphql schema and that will happen if the queries get complicated. We can surely edit the code and have sepearte schemas if we need it in the coming releases